CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+-
+
+## 0.1 - 2018-06-16
+
+The first release of Exonum Java Binding.
+
+[Unreleased]: https://github.com/exonum/exonum-java-binding/compare/v0.1...HEAD
+
+## 0.1.2 - 2018-07-19
+
+Parent module and BOM module were released as they are required dependencies to define a Java service.
+
+## 0.2 - 2018-07-23
```

Comment:
:thinking: I wonder if we shall link to releases

Reply:
Not sure what would be of most value. We can link to a release (e.g. https://github.com/exonum/exonum-java-binding/releases/tag/v0.2), but it just contains a changelog for that release; or to a commit range — but end-users don't care much about the commit history.

Reply:
Here https://keepachangelog.com/en/1.0.0/ they only posted commit ranges, no links to releases. I've added commit ranges in https://github.com/exonum/exonum-java-binding/pull/377/commits/3f6827ee94141444154abc1b0e7c2074a5224dd9 .